src/resmom/stage_func.c

```diff
@@ -1749,6 +1783,14 @@ sys_copy(int dir, int rmtflg, char *owner, char *src, struct rqfpair *pair, int
 			/* remote, try scp */
 		} else if (pbs_conf.scp_path != NULL && (loop % 2) == 1) {
 			ag0 = pbs_conf.scp_path;
+			struct stat local_sb = { 0 };
+			char ag0_cpy[MAXPATHLEN + 1] = { '\0' };
+			strncpy(ag0_cpy, ag0, sizeof(ag0_cpy));
+			if (stat_uncpath(ag0_cpy, &local_sb) == -1) {
+				log_errf(-1, __func__, "%s", ag0_cpy);
```

Comment:
I think you may have to use "errno" here, instead of -1.
https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/stat-functions?view=vs-2019